apps/web/utils/actions/group.ts

```
@@ -12,33 +13,118 @@ import {
 import { findNewsletters } from "@/utils/ai/group/find-newsletters";
 import { findReceipts } from "@/utils/ai/group/find-receipts";
 import { getGmailClient, getGmailAccessToken } from "@/utils/gmail/client";
-import { GroupItemType } from "@prisma/client";
+import { GroupItemType, Prisma, User } from "@prisma/client";
```

Biome

[error] Some named imports are only used as types.

This import is only used as a type.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)